selftests/unit/junit-4.xsd

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
```

Comment:
It came from here:
https://svn.jenkins-ci.org/trunk/hudson/dtkit/dtkit-format/dtkit-junit-model/src/main/resources/com/thalesgroup/dtkit/junit/model/xsd/junit-4.xsd

Only licensing I see is this one:
https://svn.jenkins-ci.org/trunk/hudson/LICENSE.txt